src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -99,6 +101,37 @@ public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
         return new HashMap<String, String>();
     }
 
+    protected String prepareSearchUrl(String type) {
+        URIBuilder url = null;
+        String midUrl = "search/";
+        try {
+            switch(type) {
+
+                case "user":
+                    midUrl = "profile/";
+                    url = new URIBuilder(this.baseUrl + midUrl + this.query);
+                    break;
+                case "question":
+                    url = new URIBuilder(this.baseUrl + midUrl);
+                    url.addParameter("q", this.query);
+                    url.addParameter("type", "question");
+                    break;
+                case "answer":
+                    url = new URIBuilder(this.baseUrl + midUrl);
+                    url.addParameter("q", this.query);
+                    url.addParameter("type", "answer");
+                    break;
+                default:
+                    url = new URIBuilder("");
+                    break;
+            }
+        } catch (URISyntaxException e) {
+            DAO.log("Invalid Url: baseUrl = " + this.baseUrl + ", mid-URL = " + midUrl + "query = " + this.query + "type = " + type);
+        }
+
+        return url.toString();
```

Comment:
it will never be,  see here https://github.com/loklak/loklak_server/pull/1315/files/b362b53adf1c87af1bfdec3a55f50956bc5ecb5c#diff-8aa5fffa7398b74b8670972ab5bd2d9bR125